### ISSUE TITLE
small typo in page on GitHub

### ISSUE THREAD

**Issue by OscarScholten** [quailjs/quail-ckeditor#7]
Hi all,

There is a small typo on the GitHub page: "Accessibility feedbacl for the CKEditor text editor.", should be "... feedbacK ...".

Cheers, Oscar
